pyclesperanto_prototype/_tier8/_utilities.py

```diff
@@ -97,7 +97,11 @@ def shear_angle_to_shear_factor(angle_in_degrees):
     -------
     float
     """
-    return 1.0 / math.tan((90 - angle_in_degrees) * math.pi / 180)
+    try:
+        shear_factor = 1/math.tan((angle_in_degrees) * math.pi / 180)
+    except ZeroDivisionError:
+        shear_factor = 0
+    return shear_factor
```

Comment:
```suggestion
    return 1.0 / math.tan((90 - angle_in_degrees) * math.pi / 180)
```

We should undo this change, because it breaks shearing (see demo/transforms/shearing.ipynb). I will make a suggestion of where to make a change that fixes deskew AND keeps shearing working ;-)